lib/livebook/hubs.ex

```diff
@@ -191,6 +191,9 @@ defmodule Livebook.Hubs do
         {:ok, _} ->
           :ok
 
+        {:error, {:already_started, _pid}} ->
+          :ok
```

Comment:
Currently, whenever a chub is updated (e.g. emoji change) we call `save_hub`, which tries to connect, and since it is connected we show an error.